app/components/post-list.tsx

```diff
@@ -43,11 +42,5 @@ export function PostTitle({
 }) {
   const Element = asElement ?? "span";
 
-  return (
-    <Element
-      dangerouslySetInnerHTML={{
-        __html: marked.parseInline(title),
-      }}
-    />
-  );
+  return <Element>{title}</Element>;
```

Comment:
Need to find a different solution for rendering mini markdown blocks without shipping the parser to the client. Perfect use case for server components, wish I could use them here 😭